supertokens_python/asyncio/__init__.py

```diff
@@ -55,12 +55,12 @@ async def get_user_count(
     include_recipe_ids: Union[None, List[str]] = None, tenant_id: Optional[str] = None
 ) -> int:
     return await Supertokens.get_instance().get_user_count(
-        include_recipe_ids, tenant_id
+        include_recipe_ids, user_context=None, tenant_id=tenant_id
```

Comment:
you should not pass None as user context. Make it an optional argument to the function (at the end of the function), and give it none by default. Add this to the changelog as well.

Reply:
This applies to all functions in this file, and also the syncio folder.